README.md

```diff
@@ -2,57 +2,317 @@
 describe the ranks and objects over time, which can be visualized
 using Paraview. Additionally, the task visualizer can produce PNGs
 directly using a VTK workflow to render a visualization of ranks and
-tasks over phases.
+tasks over phases (as seen below).
 
 ![Example Output PNG](./docs/example-output-image.png)
 
-## Building the Python bindings
+## Getting Started
 
-### Requirements
+You will need the following dependencies:
 
-In order to build the python bindings, make sure you have a Python <ins>`3.8` or `3.9`</ins> environment, with the `nanobind` package installed. You can install `nanobind` with `pip`:
+1. A C++ compiler that supports C++17
+2. [`cmake`](https://cmake.org/cmake/help/latest/) >= 3.17
+3. [`VTK`](https://docs.vtk.org/en/latest/index.html) (build instructions [here](https://gitlab.kitware.com/vtk/vtk/-/blob/master/Documentation/docs/build_instructions/build.md))
 
-```bash
-pip install nanobind
+Begin by cloning `vt-tv`:
+
+```
+git clone https://github.com/DARMA-tasking/vt-tv.git
+```
+
+_In future directions, we will assume that the `vt-tv` source is located in `${VTTV_SOURCE_DIR}`._
+
+## Installation and Usage
+
+`vt-tv` can installed as either a standalone C++ app or as a Python module. Instructions for both cases are included in the dropdowns below.
+
+<details>
+<summary><b>Standalone</b></summary>
+
+### 1. Build
+
+For the simplest build, run from `${VTTV_SOURCE_DIR}`:
+
+```
+VTK_DIR=/path/to/vtk/build ./build.sh
+```
+
+To build and run tests, add the `--tests-run` flag:
+
+```
+VTK_DIR=/path/to/vtk/build ./build.sh --tests-run
+```
+
+More documentation for `build.sh` can be found within the script itself, including examples.
+
+Alternatively, for an interactive build process, run:
+
+```
+./interactive_build.sh
 ```
 
-You must have a C++ compiler that supports C++17, and `cmake` >= 3.17.
+_In future directions, we will assume  that the `vt-tv` build is in `${VTTV_BUILD_DIR}`._
 
-Finally, you must have a (<ins>C++</ins>) [VTK](https://vtk.org/) build available on your system. We recommend building from source, and the currently tested version is `9.3.0`. You can find instructions for building VTK [here](https://gitlab.kitware.com/vtk/vtk/-/blob/master/Documentation/docs/build_instructions/build.md).
+---
 
-### Building
+### 2. Usage
 
-To build the python bindings, you must specify in the `VTK_DIR` environment variable the path to the VTK build directory:
+`vt-tv` requires two inputs:
+
+1. One or more JSON data files
+2. A YAML configuration file (which contains the path to the JSON data files)
+
+The basic call to `vt-tv` is:
 
 ```bash
-export VTK_DIR=/path/to/vtk/build
+${VTTV_BUILD_DIR}/apps/vt_standalone -c path/to/config
+```
+
+_**IMPORTANT:** The_ `path/to/config` _argument should be relative to_ `${VTTV_SOURCE_DIR}` _(see example below)._
+
+#### YAML Input
+
+A sample YAML configuration file can be found in `${VTTV_SOURCE_DIR}/config/conf.yaml`. To use it, run
```

Comment:
`A YAML configuration exemplar` may be clearer, as `sample` primarily means _a representative part or a single item from a larger whole or group_.

Reply:
Fixed